app/views/admin/accounting/transactions/_modal.slim

```diff
@@ -1,4 +1,3 @@
 .modal.wider id="transaction-modal" tabindex="-1" role="dialog" aria-labelledby="Transaction Modal" data-backdrop="static"
   div.modal-dialog
     div.modal-content
-      = render "admin/accounting/transactions/modal_content"
```

Comment:
Don't need this b/c modal content always loaded via ajax. Removing it reduces necessary ivars on main render.